db/migrate/20210323145018_add_email_tracking_to_school.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddEmailTrackingToSchool < ActiveRecord::Migration[6.1]
+  def change
+    change_table :schools, bulk: true do |t|
```

Comment:
We will also need to record the ID we get from notify

Reply:
Sure! Thought about adding a notify status too.

Reply:
That would be good

Reply:
I'm sending a unique reference to notify when I send the email. The reference is the same as the token